File: 3.py
name = input("\nEnter your name: ")
name = name.strip()

if name.title() == "Bob" or name.title() == "Alice":
    print("\n\nWelcome back, " + name.title() + "!\n\n")
else:
    print("\n\nERROR: Unknown user\n\n\t\t\tUser " + name + " not found in the system.\n\n")

File: 3.java
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {
		
		Scanner scanner = new Scanner(System.in);
		
		System.out.print("Enter your name: ");
		String name = scanner.nextLine();
		name = name.trim().toLowerCase();
		
		if (name.equals("bob") || name.equals("alice")) 
			System.out.println("\n\nWelcome back, " + name + "!");
		else { System.out.println("\n\nUSER NOT FOUND"); }
	}

}
